example/src/classes/util/input-field/input-field-builder.js

```diff
@@ -0,0 +1,68 @@
+import './input-field.css';
+import DefaultBuilder from '../default-builder';
+
+const InputFieldCssClasses = {
+    CONTAINER: 'field__container',
+    CONTAINER_REVERSE: 'field__container_reverse',
+};
+
+/**
+ * Пример наследования классов.
+ * Данная реализация строителя, унаследованная от базового строителя, переопределяет
+ * два метода родительского класса setTextContent и reset, а так же добавляет новый
+ * собственный метод setKeyUpCallback.
+ */
+export default class InputFieldBuilder extends DefaultBuilder {
```

Comment:
We should create InputComponent that extends base component and have those methods for working with input.